package.json

```diff
@@ -40,17 +40,17 @@
     "all": true
   },
   "scripts": {
-    "pack": "$(npm bin)/lerna exec -- npm pack",
```

Comment:
Can you please also fix the other `package.json` files in the repo? Should be a few in `src` and one in `test`.